apps/prairielearn/src/ee/pages/administratorInstitutionGeneral/administratorInstitutionGeneral.ts

```diff
@@ -75,7 +75,7 @@ router.post(
           row_id: req.params.institution_id,
         });
       });
-      flash('success', 'Successfully updated enrollment limits.');
+      flash('success', 'Successfully updated general information and enrollment limits.');
```

Comment:
Should we just be really general?

```suggestion
      flash('success', 'Successfully updated institution settings.');
```